Bottom: 9fdb1c4ce072fef5c70660420a0d46c1004c5d7c
Top:    1f76609aa8aa4120f7fe73b2989a93125320762c
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-05-29 14:49:13 -0700

Refresh of rstream-snprint

---

diff --git a/examples/rstream.c b/examples/rstream.c
index 83f9404..054d11e 100644
--- a/examples/rstream.c
+++ b/examples/rstream.c
@@ -136,13 +136,13 @@ static void size_str(char *str, size_t ssize, long long size)
 static void cnt_str(char *str, size_t ssize, long long cnt)
 {
 	if (cnt >= 1000000000)
-		sprintf(str, ssize, "%lldb", cnt / 1000000000);
+		snprintf(str, ssize, "%lldb", cnt / 1000000000);
 	else if (cnt >= 1000000)
-		sprintf(str, ssize, "%lldm", cnt / 1000000);
+		snprintf(str, ssize, "%lldm", cnt / 1000000);
 	else if (cnt >= 1000)
-		sprintf(str, ssize, "%lldk", cnt / 1000);
+		snprintf(str, ssize, "%lldk", cnt / 1000);
 	else
-		sprintf(str, ssize, "%lld", cnt);
+		snprintf(str, ssize, "%lld", cnt);
 }
 
 static void show_perf(void)
